docs/extending/admin_views.md

```diff
@@ -400,6 +400,16 @@ Image usage
 {% url 'wagtailimages:image_usage' image_id %}
 ```
 
+### Profile Picture
+
+To get the wagtail user profile picture you can use the `avatar_url` template tag.
```

Comment:
Ok. Maybe just change to 'current', the main issue was that Wagtail should be capitalised. However, it's best to find a different way to word instead of referring to Wagtail when possible.